CHANGELOG.md

```diff
@@ -61,6 +61,10 @@ Other breaking changes:
 * [BUGFIX] smartmon: Escape double quotes in device model family #772
 * [BUGFIX] Fix log level regression in #533 #815
 * [BUGFIX] Correct the ClocksPerSec scaling factor on Darwin #846
+* [BUGFIX] Count core throttles per core and per package #871
+* [BUGFIX] Fix netdev collector for linux #890
+* [BUGFIX] Fix memory corruption when number of filesystems > 16 #900
```

Comment:
On FreeBSD. Let's not worry people too much :)

Reply:
True, will update.